### PR TITLE
Add policy text

### PR DESCRIPTION
**Pull request by rnicoll**
Add the latest copy of the proposal text from Google Drive, to the repository readme. Credit to Marshall & Patrick for writing this text, I'm just transcribing it.

Marked as draft with intent that we can either confirm the text and remove the draft label from this commit, or merge this and then any revisions later are clearly version tracked.

### CODE REVIEW

File: README.md
Comment (chromatic) [inline README.md:22]: I'm little-c chromatic; you might be thinking of my father, Big-C Chromatic :zany_face:

Comment (rnicoll) [reply README.md]: :D Fixed

**Comment by rnicoll** [comment]
> Text is good.
> 
> Would strongly recommend to put it in a separate file from `README.md`, so that the latter can later be adapted to show summarized output from further on the process, like the actual address, without changing the checksum of the policy file.
> 
>     * `README.md`: description of this repository and general info, could summarize other files.
> 
>     * `POLICY.md`: static description of the current payout policy and surrounding rules. Policy changes would simply create a new policy document and archive this document. Preventing overwrites => prevent any possible confusion as to whom agreed to what and when. I think this is critical because it discourages and reduces the effectiveness of any future misinformation.

Split it into a new file now.

File: PROPOSAL.md
Comment (chromatic) [inline PROPOSAL.md:22]: I agree in spirit with this, however I can imagine the possibility of a hotfix release containing only one or two commits (suppose that we discover a crash bug on startup on one of our platforms and can't find a workaround). I would not want to distribute 500k Doge to the maintainers who are probably the ones who had to create that commit because we're the ones who let that bug through.

I'm not sure what alternate wording might work better, though "Major or Minor release" might be fine; we could consider the scenario I proposed a "Patch release".

Comment (rnicoll) [reply PROPOSAL.md:22]: I'm happy with/without threshold criteria, and 25 PR cut-off seems as good a line as any if we want a preset cut-off. Hoping to get comments from chromatic and Marshall on this before I update the text, though.

Comment (chromatic) [reply PROPOSAL.md:22]: Patrick's suggestion makes sense to me, and it catches the case where a patch release contains commits we want to credit to contributors for distribution purposes.

When we assign weights to commits, we can (and have in the past) exclude maintainer commits where we should have known better.

Comment (rnicoll) [reply PROPOSAL.md]: Revised the text on that basis - I'll squash the commits down if there's general agreement, but wanted to keep the amendment in its own commit so it's clear what I've changed.

File: PROPOSAL.md
Comment (chromatic) [inline PROPOSAL.md:23]: Maybe add "credited in the release notes *since the previous major or minor release*" to make our desired intent completely clear?

File: PROPOSAL.md
Comment (michilumin) [reply PROPOSAL.md]: Yes I think this is good as long as it remains criteria for payout roubd grouping and not determining what a release type is. Maybe some addendum that a patch release won't trigger a payout if say for some reason a 26th PR is raised at the last minute? Just some kind of edge case catcher; that 26th PR could be a whitespace revision for example...

File: PROPOSAL.md
Comment (rnicoll) [reply PROPOSAL.md]: Done

File: PROPOSAL.md
Comment (rnicoll) [reply PROPOSAL.md]: Not... sure how to usefully quantify that, I think generally there has to be a certain amount of applied sense. Sample text or even better a PR against this PR would be appreciated, though.

**Review by marshallhayner — APPROVED** [review]

**Review by marshallhayner — APPROVED** [review]

**Review by marshallhayner — APPROVED** [review]

**Review by marshallhayner — APPROVED** [review]

**Review by marshallhayner — APPROVED** [review]

**Review by marshallhayner — APPROVED** [review]

**Review by marshallhayner — APPROVED** [review]

**Review by marshallhayner — APPROVED** [review]

File: PROPOSAL.md
Comment (rnicoll) [reply PROPOSAL.md]: I'm generally leaning toward "We cannot perfectly quantify this, and trying will make us crazy", but if there's a suggested number or revised text do let me know.

**Comment by rnicoll** [comment]
Squashed the commits so far and pushed. Potentially can address the concern about hostile takeover by increasing it to 3 months?

Generally; appreciate suggested text where possible, as it's often tricky to see how to capture specific concerns in text.

**Comment by rnicoll** [comment]
Sorry about all the pushes there, was trying to keep the commit verification history but as one of the commits was made on GitHub, it doesn't have the signing key. Squash down to one now.

Suggest we merge this as-is, conscious there's a question of terms on PR significance, but I'm not sure there's specific text that would cover this (there may have to be an element of judgement). This is still marked draft, so intent would be any further tweaks are handled as PRs against this file. Is everyone okay with that?

**Comment by rnicoll** [comment]
Added some text to address pull request significance, so any whitespace-only/comment-only PR wouldn't count. Let me know if that's okay and/or means the 25 figure should be adjusted.

I've also remove the draft tag, with intent we finalise details here before merging.

**Comment by rnicoll** [comment]
For clarity; acknowledging the request not to merge until we've addressed all questions.

**Comment by rnicoll** [comment]
For avoidance of doubt, is this okay to merge now then?

**Comment by rnicoll** [comment]
Added the suggested changes, squashed and pushed.

**Comment by rnicoll** [comment]
Revised as policy, will give others a chance to review before merging.

For reference I get a SHA256 hash of a1fd0f7c92a52a519ac922a2285c8add27ae370c4506aefd2143a1697d2c4b2e with CRLF line endings.

**Pull request merged**
